### PR TITLE
Feature/board of director ali

### PR DESCRIPTION
**Pull request by aliassan**
# Description

Add Expense Account to BoD.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# Checklist:

* **Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Comment by farrellh1** [comment]
I think you need to handle in the expense tab where every interaction that had modifier onlyOwner (which means only bod can call) will add an action
you can see it on the bank tab, I handled it. for example, in bank, the flow will be like

**BOD**
- click transfer/pause/transfer ownership/push tip will open up a modal (maybe add a red text to info the bod this will create a bod action)
- add all the fields that are required for the bank interaction + u need to add a description because in bod action we need a description of why we do this such an action
- create an action to the body contract, encode the function signature, etc
- on success hit backend endpoint create action

**Comment by aliassan** [comment]
> I think you need to handle in the expense tab where every interaction that had modifier onlyOwner (which means only bod can call) will add an action you can see it on the bank tab, I handled it. for example, in bank, the flow will be like
> 
> **BOD**
> 
> * click transfer/pause/transfer ownership/push tip will open up a modal (maybe add a red text to info the bod this will create a bod action)
> * add all the fields that are required for the bank interaction + u need to add a description because in bod action we need a description of why we do this such an action
> * create an action to the body contract, encode the function signature, etc
> * on success hit backend endpoint create action

Okay, this PR was to add expense account to the BOD tab, I'll create a separate PR for the expense account tab specifically

**Review by farrellh1 — APPROVED** [review]
alright, from me all good and i tested it and it works fine. but why do u comment some of the codes?

**Comment by aliassan** [comment]
> alright, from me all good and i tested it and it works fine. but why do u comment some of the codes?

I commented out code for the Add Action modal because it was causing test coverage to fail and we're not using it in the UX for adding actions we're the existing UI of Bank Section and Expense Section so it looks like it's not necessary

File: app/src/components/sections/SingleTeamView/forms/AddActionForm.vue
Comment (hermannleboss) [inline app/src/components/sections/SingleTeamView/forms/AddActionForm.vue:1]: We was planing to stop using the form add action. 
So no need to use this form again.

**Review by hermannleboss — APPROVED** [review]
LGTM

**Pull request merged**